.gitignore

```diff
@@ -57,11 +58,30 @@ target/
 
 # IDE
 .idea/
+.DS_Store
+
+#App files
+wger/config/
+wger/core/migrations
+wger/email/migrations
+wger/exercises/migrations/0004_auto_20190606_1001.py
+wger/exercises/views/exercises_special.py
+wger/gym/migrations/0007_auto_20190606_1001.py
+wger/manager/migrations/0008_auto_20190606_1001.py
+wger/nutrition/migrations/0002_auto_20190606_1001.py
+wger/settings.py
+wger/settings.py--no-start-server
+package-lock.json
+package.json
+
 
 # External Libraries 
 wger/core/static/bower_components
 bower_components/
 node_modules
-.env
-.vscode/
-venv/
\ No newline at end of file
+
+#Compiled files
+wger/exercises/migrations/0004_auto_20190606_2122.py
+wger/gym/migrations/0007_auto_20190606_2122.py
+wger/manager/migrations/0008_auto_20190606_2122.py
+wger/nutrition/migrations/0002_auto_20190606_2122.py
```

Comment:
Still, at this place, I see migrations in the .ignore file

Reply:
Thanks for the good eye. I have removed them.

Reply:
As you have added a test for all special exersice_infos, kindly include even the one of a single special exercise